easybuild/easyconfigs/b/Bader/Bader-1.04-intel-2020b.eb

```
@@ -0,0 +1,26 @@
+easyblock = 'MakeCp'
+
+name = 'Bader'
+version = '1.04'
+
+homepage = 'http://theory.cm.utexas.edu/henkelman/code/bader/'
+description = "A fast algorithm for doing Bader's analysis on a charge density grid."
+
+toolchain = {'name': 'intel', 'version': '2020b'}
+
+source_urls = ['http://theory.cm.utexas.edu/henkelman/code/bader/download/v%(version)s']
+sources = [{'download_filename': 'bader.tar.gz', 'filename': SOURCE_TAR_GZ}]
+checksums = ['f20a0a021157d911bea06666911763b737c4ff38b39e793b8560f940fe391b8e']
+
+prebuildopts = 'sed -i "s|FFLAGS = -O2||" makefile.lnx_ifort && sed -i "s/LINK = -static/LINK = -static-intel/g" makefile.lnx_ifort && '
+buildopts = '-f makefile.lnx_ifort'
+parallel = 1
+
+files_to_copy = [(['bader'], 'bin')]
+
+sanity_check_paths = {
+    'files': ['bin/bader'],
+    'dirs': [],
+}
+
```

would be nice to add a sanity check command:
```suggestion

sanity_check_commands = ["%(name)s -h"]

```